docs/guides/automations.md

```diff
@@ -0,0 +1,349 @@
+---
+description: Prefect walkthrough on how to use automations and common best practices 
+tags:
+    - automations
+    - event-driven
+    - trigger
+title: Automations and Common Use Cases
+search:
+  boost: 2
+---
+
+# Using Automations for Dynamic Responses
+
+From the [Automations tutorial](https://docs.prefect.io/2.14.2/concepts/automations/), we were able to see the capabilities of what an automation can do and how to configure them within the UI. 
+
+In this guide, we will showcase common usecases where automations can come in handy by:
+- Creating a simple notification automation in just a few UI clicks
+- Build upon an event based automation
+- Combine into a multi-layered responsive deployment pattern
+
+!!! cloud-ad "Available only on Prefect Cloud"
+        Automations are only available on Prefect Cloud, please refer to the [Cloud documentation](https://docs.prefect.io/2.13.4/cloud/#welcome-to-prefect-cloud) to see what 
+        additional features are available such as Events and webhooks!
+
+
+## Creating the test script
+
+Automations are the one stop shop to react to different workflows by responding to specific actions recorded by Prefect. 
+
```

Comment:
I'd add a prerequisites section here - some other guides that have step-by-step instructions have and it can be good for users.